icaruscode/CRT/CRTUtils/CRTT0MatchAlg.h

```diff
@@ -59,9 +59,31 @@
 #include "TVector3.h"
 #include "TGeoManager.h"
 
+//New
+#include "larreco/RecoAlg/Cluster3DAlgs/Cluster3D.h"
+#include "Eigen/Core"
+#include "Eigen/Dense"
+#include "Eigen/Eigenvalues"
+#include "Eigen/Geometry"
+#include "Eigen/Jacobi"
+
+
 
 namespace icarus{
 
+  struct tpc_cand {
+
+	bool found_match = false;
+	recob::Track thistrack;
+	TVector3 PCA_start_pos{DBL_MAX,DBL_MAX,DBL_MAX};
+	TVector3 PCA_start_dir{DBL_MAX,DBL_MAX,DBL_MAX};
+	TVector3 PCA_end_pos{DBL_MAX,DBL_MAX,DBL_MAX};
+	TVector3 PCA_end_dir{DBL_MAX,DBL_MAX,DBL_MAX};
+	int start0_end1_whichbest = -1;
```

Comment:
L82-83: It's not clear to me what these variables represent. I suggest using more descriptive names.